tables/sr-cyr.tbl

```diff
@@ -0,0 +1,21 @@
+#-index-name: Serbian-Cyrillic
+#-display-name: Serbian Cyrillic braille
+
+#+language:sr
+#+region:sr-cyr
+#+type:literary
+#+grade:1
+
+# TODO: Please correct the metadata above. It is not meant to be
+# accurate nor complete. It hasn't been verified by the table
+# author yet. It is merely an attempt by the liblouis maintainers
+# to get some sensible initial values in place.
```

Comment:
Please don't add this comment. It is a comment I as a maintainer added in various tables, but you are able to actually validate the metadata and correct it if needed.

Reply:
I dropped the comment, also from the other table.